hash.c

```diff
@@ -303,6 +303,19 @@ objid_hash(VALUE obj)
  *
  * Certain core classes such as Integer use built-in hash calculations and
  * do not call the #hash method when used as a hash key.
+ *
+ * If you're implementing a #hash value method for a compound set of values in
+ * your own classes, best practice is to combine the class and the values by
```

Comment:
This wording seems rather involved, especially "a compound set of values".
I'd suggest to go straight to the point, with something like `When implementing #hash based on multiple values, the best practice is:`